db/pgstorage/pgstorage.go

```diff
@@ -260,6 +260,9 @@ func (p *PostgresStorage) AddForcedBatch(ctx context.Context, forcedBatch *ether
 func (p *PostgresStorage) AddTrustedGlobalExitRoot(ctx context.Context, trustedExitRoot *etherman.GlobalExitRoot, dbTx pgx.Tx) error {
 	const addTrustedGerSQL = "INSERT INTO syncv2.exit_root (global_exit_root_num, global_exit_root, exit_roots) VALUES ($1, $2, $3)"
 	_, err := p.getExecQuerier(dbTx).Exec(ctx, addTrustedGerSQL, trustedExitRoot.GlobalExitRootNum.String(), trustedExitRoot.GlobalExitRoot, pq.Array([][]byte{trustedExitRoot.ExitRoots[0][:], trustedExitRoot.ExitRoots[1][:]}))
+	if err != nil && strings.Contains(err.Error(), "duplicate key value violates unique constraint") {
```

Comment:
maybe we can create an `ErrDuplicatedKey` to help with this condition, not sure if `pgx` already have something like `ErrNoRows` for this case.